bib/american.bib

```diff
@@ -3348,7 +3348,7 @@ @BOOK(
 	year = 1846,
 	url = {http://hdl.loc.gov/loc.music/sm1846.060960},
 	keywords = {music | dance description | dance description: short | polka | quadrille | quadrille: london polka quadrille},
-	filename = {/Short descriptions/19th century/Stephen Glover/[1846, us] Stephen Glover - The London Polka Quadrilles (Lee & Walker copy).pdf},
+	filename = {/Short descriptions/19th century/Stephen Glover/[1846, us] Stephen Glover - The London Polka Quadrilles (Lee {\&} Walker copy).pdf},
```

Comment:
I'm not sure if this will be unescaped in web-interface properly.
There is a validation routine that should be altered. Here is the place for configuring it (should be changes in all four configs though):
https://github.com/georgthegreat/dancebooks-bibtex/blob/master/configs/dancebooks.production.conf#L67

I'll fix this by myself.